core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsBackupResponse.java

```diff
@@ -51,6 +53,7 @@ public JGroupsBackupResponse(Map<XSiteBackup, Future<Response>> syncBackupCalls,
    public void waitForBackupToFinish() throws Exception {
       long deductFromTimeout = timeService.timeDuration(sendTimeNanos, MILLISECONDS);
       errors = new HashMap<>(syncBackupCalls.size());
+      values = new HashMap<>(syncBackupCalls.size());
```

Comment:
I wonder if it's necessary to allocate these maps even if there are no errors/values.